app/commands/git/sync_authors_and_contributors.rb

```diff
@@ -3,7 +3,7 @@ class SyncAuthorsAndContributors
     include Mandate
 
     def call
-      ::Exercise.all.each do |exercise|
+      ::Exercise.all.find_each do |exercise|
```

Comment:
Am I correct that `find_each` performs better than `each` here?

Reply:
Yeah, but without the `all`.

So `Exercise.all.each` does:
- `Exercise.all` (get all the exercises into an array)
- `.each` (enum them)

`Exercise.find_each ` does:
- Paginate through the DB records in batches of 1000 per time.

`Exercise.all.find_each` does different things depending on the version of Rails and whether `.all` returns an array or an AR Relationship

Reply:
Aha!